sht-common/sht.h

```diff
@@ -96,9 +96,27 @@ s8 sht_measure(void);
  */
 s8 sht_read(s32 *temperature, s32 *humidity);
 
+/**
+ * Enable or disable the SHT's sleep mode between measurements, if supported.
+ * Sleep mode is enabled by default when supported.
+ *
+ * @param disable_sleep 1 to disable sleeping between measurements
```

Comment:
should also mention that 0 enables sleeping between measurements in the `@param` documentation.

And its actually `0` to enable `> 0` to disable

Reply:
fixed